tracing-subscriber/src/registry/extensions.rs

```diff
@@ -113,14 +113,16 @@ impl<'a> ExtensionsMut<'a> {
 pub(crate) struct ExtensionsInner {
     // If extensions are never used, no need to carry around an empty HashMap.
     // That's 3 words. Instead, this is only 1 word.
```

Comment:
I think this comment is removable now due to `sharded-slab`'s Pool?

Reply:
ah yup i forgot to remove the comment